components/brave_ads/common/notification_ads_features.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_COMMON_NOTIFICATION_ADS_FEATURES_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_COMMON_NOTIFICATION_ADS_FEATURES_H_
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+#include "brave/components/brave_ads/common/notification_ads_constants.h"
+
+namespace brave_ads::features {
+
+BASE_DECLARE_FEATURE(kAdNotifications);
```

Comment:
`kNotificationAds` or does this conflict with another feature?

Reply:
yes, it conflicts with another feature.